### PR TITLE
Feature/plcn 400 setup protected routes

### PR DESCRIPTION
**Pull request by s-bogelund**
Used tools available with Apollo to handle user state, namely useReactiveVar and makeVar. All the logic is stored in the UserStore file.

Some of the functionality should maybe be split into a few more files, especially if we need to add additional logic.

### CODE REVIEW

File: src/components/login/LoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import {
```
Comment (AntonEsbjergIT-Minds) [inline src/components/login/LoginForm.tsx:1]: Should this PR not be a draft until 401 is merged? seems like you have duplicated code.

Comment (s-bogelund) [reply src/components/login/LoginForm.tsx:1]: True dat. Has been changed :)

File: src/Content.tsx [hunk 2]
```diff
@@ -1,31 +1,53 @@
...
 import Contacts from './pages/Contacts';
+import LoginPage from './pages/Login';
 import NotFound404 from './pages/NotFound404';
 import Recommendations from './pages/Recommendations';
 import Suppliers from './pages/Suppliers';
 import WallOfClients from './pages/WallOfClients';
 
 const Content = () => {
+	const currentUser = UserStore.useLoginState();
+	console.log('currentUser', currentUser);
```
Comment (itm-mne) [inline src/Content.tsx:18]: Remove clog

File: src/Content.tsx [hunk 3]
```diff
@@ -1,31 +1,53 @@
...
+						<Route path="/contacts" element={<Contacts />} />
+						<Route path="/suppliers" element={<Suppliers />} />
+						<Route path="/recommendations" element={<Recommendations />} />
+						{currentUser.role === 'admin' && (
+							<Route
+								path="/create-user"
+								element={
+									<PageContainer>
+										<Typography variant="h5">
+											Here you should hopefully be able to create a new user :)
```
Comment (itm-mne) [inline src/Content.tsx:35]: Maybe add TODO here to show what you wanna do with it.

**Review by itm-mne — CHANGES_REQUESTED** [review]
Minor cleanup and suggestions for use of Access-wrapper on login-page. Also make sure to resolve conflicts with `dev`.

File: src/Content.tsx (hunk 3 above)
Comment (s-bogelund) [reply src/Content.tsx:35]: Added in other branch

File: src/pages/Login.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+						display: 'flex',
+						flexDirection: 'column',
+						alignItems: 'center',
+						width: '100%',
+					}}
+				>
+					<LoginForm failedLogin={failedLogin} />
+				</Box>
+			</PageContainer>
+		</Access>
```
[the remark this answers is not in the document]
Comment (s-bogelund) [reply src/pages/Login.tsx:34]: Will create seperate issue

**Pull request merged**